cmake/deploy-osx.cmake.in

```diff
@@ -0,0 +1,15 @@
+set(MACDEPLOYQT "@MACDEPLOYQT@")
```

Comment:
Is this only for adding the qt libs/plugins or does this handle rpaths to? 
What about the rpath for other libs? does cmake fix that with the new settings? 
Did we not have some other code to install the qt libs, is that still active?

Reply:
> Is this only for adding the qt libs/plugins or does this handle rpaths to?

It copies qt libs/plugins and adjusts the RPATHs to those for all the libraries
> What about the rpath for other libs? does cmake fix that with the new settings?

Yes, with the new settings the RPATH of all binaries are set to @loader_path and @loader_path/../Frameworks when packaged

> Did we not have some other code to install the qt libs, is that still active?

Yes, ivw_deploy_qt and ivw_qt_add_to_install 
**ivw_deploy_qt** is only active on Windows to be able to run inviwo outside of the dev environment. We might want to refactor this.

**ivw_qt_add_to_install** is probably not needed on mac/windows anymore. I have not tried to remove it. On linux we would need to use an open source tool (I think that it si called linuxdeployqt or something).

Reply:
https://github.com/inviwo/inviwo/blob/f37dcbdfda16646529283ae007339c3b95a447f8/cmake/installutils.cmake#L279

seems like that might do duplicated work then? should we use the same system on windows?

Reply:
might as well use windeployqt on windows, and get rid of that crazy code to find the plugins in ivw_qt_add_to_install?

Reply:
I am testing it on Mac now but don’t have a windows machine right now. Will have to hope that the CI build captures any errors :)

since we already run windeployqt after each build I think that it should be possible to remove all of the stuff, but otherwise it will probably work if we just run it again during packaging

Reply:
Works on Windows now as well. You should checkout and test it :)
We might want to filter out which dlls to run windeployqt on, e.g., only inviwo-module dlls... However, it seems to work now so perhaps no need to mess with it more.